decidim-accountability/app/forms/decidim/accountability/admin/result_form.rb

```diff
@@ -38,12 +38,25 @@ def proposals
           @proposals ||= Decidim.find_resource_manifest(:proposals).try(:resource_scope, context.current_feature)&.order(title: :asc)&.pluck(:title, :id)
         end
 
-        def organization_scopes
-          current_organization.scopes
+        def participatory_space_scope
```

Comment:
We have the `HasScope` concern, but would not apply here since those are form objects, and the already existing concern is for models. 

I'm 👍  on using a concern for this, but we have to be careful with the naming, and have some good docs warning developers this concern should only be used for form objects.

Reply:
Well, I'm not sure if this should be a concern. All that forms have a scope and a category, and updating all every time you change one is tedious, but maybe creating concerns in the core engine for forms defined in modules increase complexity and reduce flexilibity. :thinking:

Reply:
Mmmmm what about moving these common methods up to `Decidim::Form`? Or maybe better, create a `Decidim::FormOnFeature` equivalent to `Decidim::Form` that assumes the availability of `current_feature` and inherit from that instead? Are those checks on `current_feature` actually needed or these forms always need a `current_feature`?